create-react-app/src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```

Comment:
Hmm, what's jest-dom doing in here? There's a @testing-library/cypress package.

Reply:
All my changes in this PR were based on cypress-documentation#3883. It would be great to know what else should be done here to create the best DX

Reply:
This comes with the CRA boilerplate